### PR TITLE
pin miniconda image and add missing dependency

### LINKED ISSUE #16: cannot build singularity/docker images with the current continuumio/miniconda3 image
Dear all,

I can't build the nflo singularity/docker image using the latest `continuumio/miniconda3` base image. So I've pinned the base image and added the missing requirement. I will open a PR

### PR DESCRIPTION
**Pull request by bunop**
close #16

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
+FROM continuumio/miniconda3:23.5.2-0 AS build
 
 LABEL authors="andrea.talenti@ed.ac.uk" \
       description="Docker image containing base requirements for nf-LO pipelines"
 
 # Install the package as normal:
 COPY environment.yml .
 
+# Add missing mamba dependency then clean cache
+RUN apt-get update && apt-get install -y \
```
Comment (RenzoTale88) [inline Dockerfile:10]:
```suggestion
RUN apt-get update -qq && apt-get install -qq -y \
```

File: singularity.def [hunk 2]
```diff
@@ -1,17 +1,23 @@
...
-authors="andrea.talenti@ed.ac.uk" 
+authors="andrea.talenti@ed.ac.uk"
 description="Docker image containing base requirements for nf-LO pipelines"
 
 %post
 # Add anaconda to path, if not already present
 export PATH=/opt/conda/bin:${PATH}
 
+# Add missing mamba dependency then clean cache
+apt-get update && apt-get install -y \
```
Comment (RenzoTale88) [inline singularity.def:16]:
```suggestion
apt-get update -qq && apt-get install -y -qq \
```

**Review by RenzoTale88 — CHANGES_REQUESTED** [review]
Thanks for this, I would make the installation of deps less verbose

**Comment by bunop** [comment]
@RenzoTale88 , here are those changes

**Review by RenzoTale88 — APPROVED** [review]

**Pull request merged**